<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="sldRail" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="MyBtn" name="btnSld">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="point1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="point2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="point3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pointRight">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyBtn</class>
   <extends>QPushButton</extends>
   <header location="global">mybtn.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
